@navikt/core/css/darkside/button.darkside.css

```diff
@@ -257,10 +257,11 @@
 
 .navds-button:where(:disabled, .navds-button--disabled) {
   cursor: not-allowed;
+  user-select: none;
```

Comment:
Why `user-select: none;` on disabled button?

Reply:
When using `<Button as="a">`, you can select the text before this change.